plaidml2/bridge/keras/__init__.py

```diff
@@ -40,6 +44,24 @@ def _prepend_name_scope(name, default):
     return r
 
 
+def _normalize_axis(axis, ndims, name=""):
+    negative_axis_given = False
+    if axis < 0:
+        axis += ndims
+        negative_axis_given = True
+    if axis < 0 or ndims <= axis:
+        if negative_axis_given:
+            axis -= ndims
+        if name:
```

Comment:
This might actually be more clear as a one-liner -- `name = 'for {} op '.format(name) if name else ''`
Or in py3, `name = f'for {name} op' if name else ''`